### DISCUSSION TITLE
Ability to remove unwanted/buggy dots from chart

### DISCUSSION THREAD

**Discussion by plastik21** [NightscoutFoundation/xDrip#2611] [Ideas]
Hello.
When I set and start a new sensor I always have erroneous dot in chart, for example: 22 mmol, while the blood glucose is about 14 mmol.
![image](https://user-images.githubusercontent.com/22352217/211173823-e8a62336-7574-4d5e-b2aa-e034d2fde287.png)

It would be helpful if there will be possibility to remove such dots from chart.

Thanks

**Comment by Navid200** [comment]
You can mark a reading as a bad reading and exclude it from statistics:
https://navid200.github.io/xDrip/docs/Display/ExcludeReading.html
